templates/juniper_junos_show_lacp_interfaces.textfsm

```diff
@@ -0,0 +1,11 @@
+Value Required,Filldown AGGREGATED_INTERFACE (\S+)
+Value List MEMBER_INTERFACE (\S+)
+Value List RECEIVE_STATE (\S+\s?\w+)
+Value List TRANSMIT_STATE (\S+\s?periodic)
+Value List MUX_STATE (\S+\s?\S*)
+
+Start
+  ^Aggre -> Continue.Record
+  ^Aggregated\sinterface:\s${AGGREGATED_INTERFACE}(\s|$$) -> Continue
+  ^\s+${MEMBER_INTERFACE}\s+${RECEIVE_STATE}\s+${TRANSMIT_STATE}\s+${MUX_STATE}(\s|$$) -> Continue
```

Comment:
Do you mind restructuring this template a bit?

I think it may be good to act off the headers within the output for both the LACP state and LACP protocol.

Something like:

```
Value Required,Filldown AGGREGATED_INTERFACE (\S+)
Value List MEMBER_INTERFACE (\S+)
Value List RECEIVE_STATE (\S+\s?\w+)
Value List TRANSMIT_STATE (\S+\s?periodic)
Value List MUX_STATE (\S+\s?\S*)

Start
  ^Aggregated -> Continue.Record
  ^Aggregated\sinterface:\s${AGGREGATED_INTERFACE}(\s|$$) -> Continue
  ^LACP\s+state
  ^\s+\S+
  ^LACP\s+protocol -> LACP_PROTO
  ^\s*$$
  ^. -> Error

LACP_PROTO
  ^\s+${MEMBER_INTERFACE}\s+${RECEIVE_STATE}\s+${TRANSMIT_STATE}\s+${MUX_STATE}(\s|$$) -> Continue
  ^Aggregated -> Start
  ^\s*$$
  ^. -> Error
```

Reply:
Hi, @FragmentedPacket 

Thank you for the good suggestions.
I added LACP protocol header in the template.
```
Start
  ^Aggregated -> Continue.Record
  ^Aggregated\sinterface:\s${AGGREGATED_INTERFACE}(\s|$$) -> Continue
  ^\s+LACP\s+state: -> Continue
  ^\s+LACP\sprotocol: -> LACP_PROTO
  ^\s*$$
  ^{master:\d+}

LACP_PROTO
  ^\s+${MEMBER_INTERFACE}\s+${RECEIVE_STATE}\s+${TRANSMIT_STATE}\s+${MUX_STATE}(\s|$$) -> Continue
  ^\s*$$
  ^Aggregated -> Continue.Record
  ^Aggregated\sinterface:\s${AGGREGATED_INTERFACE}(\s|$$) -> Start
```